lib/Constants.js

```diff
@@ -31,7 +31,11 @@ function searchForHome() {
 }
 
 function getRegistry() {
-    if (process.env.ADAPT_REGISTRY) return process.env.ADAPT_REGISTRY;
```

Comment:
Why did you drop the environment variable?

Reply:
Restored